src/sdl/sdlgraphics.cpp

```diff
@@ -432,8 +429,8 @@ namespace gcn
               if(SDL_BYTEORDER == SDL_BIG_ENDIAN) {
                   for (;y1 <= y2; ++y1)
                   {
-                      p[0] = (pixel >> 16) & 0xff;
-                      p[1] = (pixel >> 8) & 0xff;
+                      p[0] = pixel >> 16 & 0xff;
+                      p[1] = pixel >> 8 & 0xff;
```

Comment:
I disagree with the warning.
Even if redundant with operator priorities, those priorities seems not necessary "known".

Reply:
Sure, I don't have any hard position on this one. I can revert these with a new commit.